src/lib/components/Swap/Summary/index.tsx

```diff
@@ -119,26 +120,28 @@ export function SummaryDialog({ onConfirm }: SummaryDialogProps) {
           <Rule />
           <DetailsColumn>
             <Column gap={0.5} ref={setDetails} css={scrollbar}>
-              <Details input={inputCurrency} output={outputCurrency} />
+              <Details trade={trade} />
             </Column>
           </DetailsColumn>
           <Estimate color="secondary">
             <Trans>Output is estimated.</Trans> {/* //@TODO(ianlapham): update with actual recieved values */}
-            {/* {swap?.minimumReceived && (
-              <Trans>
-                You will receive at least {swap.minimumReceived} {output.token.symbol} or the transaction will revert.
-              </Trans>
-            )}
-            {swap?.maximumSent && (
-              <Trans>
-                You will send at most {swap.maximumSent} {input.token.symbol} or the transaction will revert.
-              </Trans>
-            )} */}
+            <Trans>
+              You will receive at least {trade.minimumAmountOut(allowedSlippage).toSignificant(6)}{' '}
+              {outputCurrency.symbol} or the transaction will revert.
+            </Trans>
+            <Trans>
+              You will send at most {trade.maximumAmountIn(allowedSlippage).toSignificant(6)} {inputCurrency.symbol} or
+              the transaction will revert.
+            </Trans>
           </Estimate>
           <ActionButton
             onClick={onConfirm}
-            onUpdate={() => confirmPrice(price)}
-            updated={price === confirmedPrice ? undefined : updated}
+            onUpdate={() => setOriginalTrade(trade)}
+            updated={
+              tradeMeaningFullyDiffers
+                ? { message: <Trans>Price updated</Trans>, action: <Trans>Accept</Trans> }
```

Comment:
This is an object, so it should either be defined outside of the component (as it was), or memoized use `useMemo`. It should not be instantiated in the prop.